GameServer.cpp

```diff
@@ -23,8 +21,7 @@ GameServer::GameServer()
 {
 	// Initialize network socket descriptor
 	hostSocket = INVALID_SOCKET;
-	numGames = 0;
-	maxNumGames = 0;

```

Comment:
Did you want to open an issue for limiting the server load? I was on the fence about not dealing with it until it became an actual problem or getting ahead of the issue.

Reply:
There was never any intention to limit server load. I don't think server load is ever going to be an issue. It might be nice to be able to monitor it though.